projects/ngx-maplibre-gl/src/public_api.ts

```diff
@@ -4,8 +4,26 @@
 
 export * from './lib/ngx-maplibre-gl.module';
 
-// Expose control component to allow custom directives
+// Expose components to allow custom directives
 export * from './lib/control/control.component';
+export * from './lib/draggable/draggable.directive';
+export * from './lib/layer/layer.component';
+export * from './lib/image/image.component';
+export * from './lib/source/vector-source.component';
+export * from './lib/source/video-source.component';
+export * from './lib/source/canvas-source.component';
+export * from './lib/source/image-source.component';
+export * from './lib/source/raster-dem-source.component';
+export * from './lib/source/raster-source.component';
+export * from './lib/source/geojson/feature.component';
+export * from './lib/marker/marker.component';
+export * from './lib/popup/popup.component';
+export * from './lib/control/fullscreen-control.directive';
+export * from './lib/control/geolocate-control.directive';
+export * from './lib/control/navigation-control.directive';
+export * from './lib/control/attribution-control.directive';
+export * from './lib/control/scale-control.directive';
+export * from './lib/markers-for-clusters/markers-for-clusters.component';
```

Comment:
Are all these exports really required? Why do we need to export all of them?

Reply:
I assume that the idea (from upstream) is to give the ability to allow custom controls and other components to extend (subclass) from these classes, and no that there's no harm anyway

Reply:
Upsteam has this for geocoder I think. But yea, it probably can't hurt...